### DISCUSSION TITLE
thoughts on flatpak?

### DISCUSSION THREAD

**Discussion by Dieterbe** [CyberShadow/aconfmgr#167] [Ideas]
seems flatpak is a fairly popular mechanism to install and more safely execute "packages". it also seems to come up when dealing with wayland related issues (it might make things like screen sharing, file choice dialogs etc easier), though my understanding is very superficial at this point.

would it make sense for aconfmgr to treat them as "first class", ie recognize them, and track relevant changes?
(personally i use traditional packages and firejail for sandboxing so it's not a big deal for me)

**Comment by bndlfm** [comment]
I'd love something like this, I'd imagine you can probably just write a shell script but it would be awesome to have it all in the same place.
